src/app/components/subtopics-list/search-section/search-section.component.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  Component,
+  ElementRef,
+  EventEmitter,
+  OnInit,
+  Output,
+  ViewChild,
+} from '@angular/core';
+import { CommonModule } from '@angular/common';
+import {
+  debounceTime,
+  distinctUntilChanged,
+  filter,
+  fromEvent,
+  map,
+} from 'rxjs';
+
+@Component({
+  selector: 'app-search-section',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './search-section.component.html',
+  styleUrl: './search-section.component.scss',
+})
+export class SearchSectionComponent implements OnInit {
+  protected searchedTitle: string = '';
+  @Output() searchedValueSent = new EventEmitter<string>();
+
+  @ViewChild('subtopicsSearchValue', { static: true })
+  subtopicsSearchValue!: ElementRef;
+
+  ngOnInit(): void {
+    fromEvent(
```

Comment:
fromEvent represents a hot Observable, which means the value is produced outside of its control ( you either subscribe at that moment, or you lose that value ). Use a subscription initialized on top of the class, `private readonly subscription = new Subscription()` and unsubscribe from the obs.